{"uuid":"6d3f6875-4779-4911-bd9d-258b07c3b585","name":"com.qa.test.layer.FormFillingTest","children":["6b0bb1f6-27bd-4a6f-96a0-deb9b21bc082","e39c3d79-09ae-4ac9-a1ab-6a9ff98b8458","1237541d-17cd-460b-9e09-9ec0284c3aee"],"befores":[{"name":"initDriver","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. \nHost info: host: 'LAPTOP-B0L2K8D2', ip: '192.168.233.109'\nBuild info: version: '4.27.0', revision: 'd6e718d134'\nSystem info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.12'\nDriver info: io.appium.java_client.android.AndroidDriver\nCommand: [null, newSession {capabilities=[Capabilities {appium:app: C:\\Raj Setup\\Mobile_Automat..., appium:automationName: UIAutomator2, appium:deviceName: Raj_AppiumTest, platformName: ANDROID}]}]\nCapabilities {appium:app: C:\\Raj Setup\\Mobile_Automat..., appium:automationName: UIAutomator2, appium:deviceName: Raj_AppiumTest, platformName: ANDROID}","trace":"org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. \nHost info: host: 'LAPTOP-B0L2K8D2', ip: '192.168.233.109'\nBuild info: version: '4.27.0', revision: 'd6e718d134'\nSystem info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.12'\nDriver info: io.appium.java_client.android.AndroidDriver\nCommand: [null, newSession {capabilities=[Capabilities {appium:app: C:\\Raj Setup\\Mobile_Automat..., appium:automationName: UIAutomator2, appium:deviceName: Raj_AppiumTest, platformName: ANDROID}]}]\nCapabilities {appium:app: C:\\Raj Setup\\Mobile_Automat..., appium:automationName: UIAutomator2, appium:deviceName: Raj_AppiumTest, platformName: ANDROID}\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:563)\r\n\tat io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:270)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:174)\r\n\tat io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:91)\r\n\tat io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:103)\r\n\tat io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:109)\r\n\tat com.qa.utils.AndroidUtils.initDriver(AndroidUtils.java:103)\r\n\tat com.qa.test.utils.BaseTest.initDriver(BaseTest.java:27)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:188)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)\r\nCaused by: java.io.UncheckedIOException: java.net.ConnectException\r\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:496)\r\n\tat io.appium.java_client.internal.filters.AppiumIdempotencyFilter.lambda$apply$0(AppiumIdempotencyFilter.java:35)\r\n\tat io.appium.java_client.internal.filters.AppiumUserAgentFilter.lambda$apply$0(AppiumUserAgentFilter.java:87)\r\n\tat org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)\r\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.lambda$executeAsync$1(JdkHttpClient.java:383)\r\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)\r\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\r\n\tat java.base/java.lang.Thread.run(Thread.java:842)\r\nCaused by: java.net.ConnectException\r\n\tat java.net.http/jdk.internal.net.http.HttpClientImpl.send(HttpClientImpl.java:574)\r\n\tat java.net.http/jdk.internal.net.http.HttpClientFacade.send(HttpClientFacade.java:123)\r\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:455)\r\n\t... 9 more\r\nCaused by: java.net.ConnectException\r\n\tat java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1055)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.checkRetryConnect(PlainHttpConnection.java:230)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$1(PlainHttpConnection.java:206)\r\n\tat java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)\r\n\tat java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)\r\n\tat java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)\r\n\tat java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)\r\n\t... 3 more\r\nCaused by: java.nio.channels.ClosedChannelException\r\n\tat java.base/sun.nio.ch.SocketChannelImpl.ensureOpen(SocketChannelImpl.java:195)\r\n\tat java.base/sun.nio.ch.SocketChannelImpl.beginConnect(SocketChannelImpl.java:760)\r\n\tat java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:848)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)\r\n\tat java.base/java.security.AccessController.doPrivileged(AccessController.java:569)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)\r\n\t... 9 more\r\n"},"stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1734187725186,"stop":1734187725682}],"afters":[],"start":1734187725146,"stop":1734187727004}